app/controllers/ratings.js

```diff
@@ -0,0 +1,17 @@
+const logger = require('../logger');
+const { transactionRatingWeet } = require('../services/ratings');
+const { RATING_WEET_SUCCESS } = require('../constants/messages');
+
+exports.ratingWeets = async (
+  { user: { id: userId }, params: { weetId }, body: { score }, weet },
```

Comment:
This is totally fine(destructuring), but it is a little bit hard to read and prone to conflicts with future changes